requirements.txt

```diff
@@ -1,4 +1,4 @@
 wagtail
 cookiecutter
-psycopg2
+# psycopg2 for use with postgres (not compatible with pypy)
```

Comment:
Let's switch to https://pypi.python.org/pypi/psycopg2cffi, it's compatible with psycopg2 and works with pypy.